### PR TITLE
f-footer@2.0.0-beta.8 ButtonList component

### PR DESCRIPTION
**Pull request by dandel10n**
### Added
- `ButtonList` component
- Small styles fixes for IE11
- Some css global variables for MenuLog styles
- `theme` mixin for picking up MenuLog styles
- `@vue/cli-plugin-babel` to the package dev dependencies, it wasn't picked up from the root monorepo level

### Changed
- Restructured AU resource file for new button list block + added empty `linkButtonList` array for other tenants

### CODE REVIEW

File: packages/f-footer/src/assets/scss/common.scss [hunk 1]
```diff
@@ -21,6 +21,20 @@ $theme: 'je' !default;
  $footer-chevronColor    : $grey--dark !default;
  $footer-socialIconColor : $grey--dark !default;
 
+ /**
+ * Global variables which need to be changed depending on the theme (TODO: move to separate component)
+ */
+ $font-family-headings       : 'Ubuntu', sans-serif;
```
Comment (ashleynolan) [inline packages/f-footer/src/assets/scss/common.scss]: This variable should already be available as part of fozzie's variables that you include? [here](https://github.com/justeat/fozzie/blob/master/src/scss/settings/_variables.scss#L60-L62)

File: packages/f-footer/src/assets/scss/common.scss [hunk 2]
```diff
@@ -21,6 +21,20 @@ $theme: 'je' !default;
...
+ $font-family-headings--ml   : 'Aspira', 'Times New Roman', Times, serif;
+
+ $font-weight-headings       : 300; // instead of browser default, bold
```
Comment (ashleynolan) [inline packages/f-footer/src/assets/scss/common.scss]: Same for this. Should only be the menulog override variables that you need to specify – the others should already exist.

File: packages/f-footer/src/components/ButtonList.vue [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+        <h2 class="c-footer-heading">
+            {{ buttonList.title }}
+        </h2>
+        <div class="c-buttonsList">
```
Comment (ashleynolan) [inline packages/f-footer/src/components/ButtonList.vue]: Would call this `c-buttonList` to match the JS variable on line 8.

**Review by ashleynolan — COMMENTED** [review]
Couple of small comments

File: packages/f-footer/src/assets/scss/common.scss (hunk 1 above)
Comment (dandel10n) [reply packages/f-footer/src/assets/scss/common.scss]: @ashleynolan Yes, I decided to add it for better visibility of what we have for `je` theme and what we rewrite for `ml`. Happy to remove if it makes more sense not to duplicate them here

Comment (ashleynolan) [reply packages/f-footer/src/assets/scss/common.scss]: Yeh, should be ok not to duplicate. Most (if not all) of the current fozzie variables will likely get pulled over into the new module when it's created anyway.

File: packages/f-footer/src/components/tests/__snapshots__/Footer.test.js.snap [hunk 4]
```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Footer should render default component markup 1`] = `
-<footer class="c-footer">
+<footer data-theme="je" class="c-footer">
```
Comment (ashleynolan) [inline packages/f-footer/src/components/tests/__snapshots__/Footer.test.js.snap:4]: Is it worth adding a snapshot to test the `ml` theme?

**Review by ashleynolan — APPROVED** [review]
Approved – could add a test to check the theme method produces the expected snapshot?

**Review by DamianMullins — APPROVED** [review]

File: packages/f-footer/src/components/tests/__snapshots__/Footer.test.js.snap (hunk 4 above)
Comment (dandel10n) [reply packages/f-footer/src/components/tests/__snapshots__/Footer.test.js.snap:4]: I am not sure about it. With the theme there are changes in css, all html markup stays the same.

Comment (ashleynolan) [reply packages/f-footer/src/components/tests/__snapshots__/Footer.test.js.snap:4]: The data-attribute should change to `ml` though in the markup? As long as the method is tested, that should be enough tbh.

**Pull request merged**